docs/README.md

```diff
@@ -1,29 +1,51 @@
 # OpenROAD Documentation
 
-This documentation is available at [https://openroad.readthedocs.io/en/latest/](https://openroad.readthedocs.io/en/latest/)
+This repo houses code and raw files to build the on-line documentation (using Sphinx)
+and manual pages using (using Pandoc)
 
-## Build locally
+This on-line documentation is available at [https://openroad.readthedocs.io/en/latest/](https://openroad.readthedocs.io/en/latest/).
 
-### Requires:
-- Python 3.x
-- Pip
-- `virtualenv`
+## Prerequisites
 
-### Install prerequisites:
+- To install pandoc, refer to this [link](https://github.com/jgm/pandoc/blob/main/INSTALL.md). `apt-get` *should* just work for Ubuntu. 
+- To install sphinx requirements, **create a virtual environment (e.g. conda/virtualenv)** and then run `pip install -r requirements.txt`.
 
-``` shell
-virtualenv .venv
-source .venv/bin/activate
```

Comment:
Is there a reason we dropped mention of venv?

Reply:
There can be many ways the user might install the python requirements and activate the environment, I can add it back in if its preferred